### PR TITLE
x11-misc/tint2: version bump, EAPI 6 bump

### PR DESCRIPTION
**Pull request by nE0sIghT**
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=590798

### CODE REVIEW

File: x11-misc/tint2/tint2-0.12.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+}
+
+pkg_postinst() {
+	gnome2_icon_cache_update
```
Comment (nE0sIghT) [inline x11-misc/tint2/tint2-0.12.12.ebuild]: I think you meant gnome2_icon_savelist

Comment (Coacher) [inline x11-misc/tint2/tint2-0.12.12.ebuild]: Yes, that one. Wrong copy-paste on my side.

**Comment by nE0sIghT** [comment]
@Coacher 

Thanks for review. Pushed update

**Comment by Amynka** [comment]
Thank you @Coacher for review and thank you @nE0sIghT for your contribution!

**Pull request closed without merging**